content/doc/book/using/slave-master-access-control.adoc

```diff
@@ -0,0 +1,484 @@
+---
+layout: section
+title: Agent to master access control
+---
+
+[[SlaveToMasterAccessControl-Whatisthis]]
```

Comment:
I'd suggest to simplify the section names, removing the common prefix
```suggestion
[[Whatisthis]]
```

Reply:
The odd section names may be a result of the wiki exporter working to help us retain working hyperlinks across redirects.  However, we'll need to review the actual `id` tags on the source page to see if the section names match with the original page.

We may also intentionally choose to remove the section tag names because the page ultimately will be quite different from the wiki page.